portfolio/frontend/src/containers/Login/Login.module.css

```diff
@@ -41,15 +41,26 @@ p {
 
 .Login {
     width: 200px;
-    background-color: blue;
     color: white;
     margin-right: 10px;
+    padding-top: 10px;
+    height: 30px;
```

Comment:
Buttons have lots of common properties/attributes, can this be factored out

Reply:
I don't believe you can do this in HTML, but will make a fix for overlapping CSS properties.

Reply:
Done. Please re-review :)